### DISCUSSION TITLE
[A-L] (2023/24) Foglio 12 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#270] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-28 131720](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/f52b490b-957b-4eeb-9787-28e8b7c6105f)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-25 162443](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/909bc635-f0ea-4d93-8cee-8c930021b7d1)

**Comment by Elia-Belli** [comment]
Dobbiamo controllare che $U,W$ siano in somma diretta, ovvero che $(U \cap W)=\underline 0$:
è evidente che non esistano vettori di $U$ proporzionali a $(1,1,1)\Rightarrow (U\cap W)=\underline 0$

Adesso ci basta mostrare che $dim\ W=1$ e $dim\ U=2$, affinchè la somma diretta generi $\mathbb{R}^3$:
il sottospazio $W$ è generato dal singolo vettore $(1,1,1)$, quindi ha $dim\ W=1$
una base del sottospazio $U$ può essere $\\{(1,0,1),(1,1,0)\\}$, qualsiasi altro vettore di $U$ è combinazione lineare di quei due, allora $dim\ U=2$

Allora, per Grassman, $dim\ (U\oplus W)=dim\ W+dim\ U=3$ ma siccome la somma di sottospazi è un sottospazio e l'unico sottospazio che ha dimensione uguale allo spazio vettoriale è quello banale, allora $\mathbb{R}^3=U\oplus W$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/754a1cb2-c50c-4582-b0b3-1edc58e271d4)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/9ac212de-dbd5-47d2-8552-51f7820af28c)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/2790937e-8166-46f6-a26c-48527247d38d)
